test/pytest/test_bridge.py

```diff
@@ -47,13 +47,16 @@ def send_open(self, payload, channel=None, **kwargs):
         self.send_json('', command='open', channel=channel, payload=payload, **kwargs)
         return channel
 
-    async def check_open(self, payload, channel=None, problem=None, **kwargs):
+    async def check_open(self, payload, channel=None, problem=None, message=None, **kwargs):
```

Comment:
This is too ad-hoc.  If you want to do this then it might make sense to either:
  1) change `problem` to `Optional[dict[str, object]]` or
  2) add `reply_keys: Optional[Dict[str, object]] = None` or so which is passed as the `kwargs` of `self.assert_msg()` for either `ready` or `close`

haven written the two out I think i prefer the second one.